crates/services/upgradable-executor/build.rs

```diff
@@ -62,9 +62,6 @@ fn build_wasm() {
         args.extend([
             "--version".to_owned(),
             crate_version.to_owned(),
-            // We can use the offline mode because it was already downloaded
-            // by the `build.rs` dependencies requirements.
-            "--offline".to_string(),
```

Comment:
Did we just not need this before? It seems like the `wasm_executor_path` is the "offline" path. Is that right?

Reply:
It was not needed, I just tried to cover all the cases of how this crate could be used. I assumed that because we will build `fuel-core-wasm-executor` as a dependency, it would pull all required crates. But it is not true because dependency may have different(bumped) dependencies, while to build a WASM bytecode, we are using `--locked.`